package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "build": "rimraf lib && yarn check-format && tsc -d -p tsconfig-src.json",
     "test": "jest",
-    "ruling": "ts-node ruling/index.ts",
+    "ruling": "ts-node --files ruling/index.ts",
```

Comment:
see https://stackoverflow.com/questions/51610583/ts-node-ignores-d-ts-files-while-tsc-successfully-compiles-the-project